lib/pages/settings/model/request.dart

```diff
@@ -1,9 +1,17 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+
 class Request {
-  Request(this.userId, this.requestBody);
+  Request(this.userId, this.requestBody, this.timestamp);
 
   /// The user who created this request
   final String userId;
 
   /// The body of the request
   final String requestBody;
+
+  /// Boolean value represented whether the request has been processed by mods
+  final bool processed = false;
+
+  /// Timestamp representing the moment when a request is sent
+  final Timestamp timestamp;
```

Comment:
Timestamp is part of the firestore API, it shouldn't be in the model. Make this LocalDateTime or just remove it entirely (you can just call Timestamp.now() from makeRequest directly, no need to pass it from the UI).